### ISSUE TITLE
listrecipient: add contact to an existing contact list

### ISSUE THREAD

**Issue by samstride** [mailjet/mailjet-apiv3-go#59]
Hi,

Ran into this bug in the go client:

``` go
var listData []resources.Listrecipient
request := &mailjet.Request{
	Resource: "listrecipient",
}
fullRequest := &mailjet.FullRequest{
	Info: request,
	Payload: resources.Listrecipient{
		IsUnsubscribed: true,
		ConactALT:      "passenger@mailjet.com",
		ListID:         11333,
	},
}
err = mailjetClient.Post(fullRequest, &listData)
if err != nil {
	log.Fatalln(err)
}
```

Returns this error:  

```
400: Object properties invalid (MJ08 Property Contact is invalid: MJ03 A non-empty value is required)
```

I switched to http.Post with body:

```json
"IsUnsubscribed": "true",
"ContactAlt":      "passenger@mailjet.com",
"ListID":         11333,
```

and it works.

Can you please fix the go client since it's easier to use and keeps things consistent for our use case?

Cheers.

**Comment by bojanmilinkovic** [comment]
Is this fixed? Since I am also getting the same error 

mr := &mailjet.Request{
		Resource: "listrecipient",
	}
	fmr := &mailjet.FullRequest{
		Info: mr,
		Payload: &resources.Listrecipient{
			IsUnsubscribed: true,
			ContactID:      167078506,
			ContactALT:     "bokitest8@gmail.com",
			ListID:         39784,
		},
	}

	err := m.client.Post(fmr, &data)
	if err != nil {
		return fmt.Errorf("Error adding contact to contact list: %w", err)
	}
	
	
	
	Response:
	Error adding contact to contact list: Unexpected server response code: 400: Object properties invalid (MJ08 Property Contact is invalid: MJ03 A non-empty value is required)

**Comment by tsidei** [comment]
fixed by #69
